packages/shared/friends/sagas.ts

```diff
@@ -1261,6 +1276,21 @@ function* handleUpdateFriendship({ payload, meta }: UpdateFriendship) {
             updateTotalFriendRequestsPayload[updateTotalFriendRequestsPayloadSelector] - 1
         }
 
+        // TODO!: remove FF validation once the new flow is the only one
+        if (newFriendRequestFlow && incoming && action === FriendshipAction.REJECTED) {
+          // Build message
+          const rejectFriendRequest = {
+            userId: getUserIdFromMatrix(userId)
+          }
+
+          // Send messsage to renderer via rpc
+          getRendererModules(store.getState())
```

Comment:
const rendererModules = yield select(getRendererModules)

Reply:
Oops sorry! I've already merged the branch into another PR. I'll address your comments in that one. Thx 🙌🏼